language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/createvar/ErrorHandleInsideCodeAction.java

```diff
@@ -53,13 +54,18 @@ public int priority() {
         return 997;
     }
 
+    @Override
+    public boolean validate(Diagnostic diagnostic, DiagBasedPositionDetails positionDetails, 
+                            CodeActionContext context) {
+
+        return CodeActionNodeValidator.validate(context.nodeAtCursor()) && 
```

Comment:
Shall we interchange the conditions. First let's check the var assignment required message.

Reply:
+1